strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/configuration/StoragesConfigurationControllerTestIT.java

```diff
@@ -279,6 +290,22 @@ public void testAddGetRepository()
         deleteRepository(storage0.getId(), r0_1.getId());
         deleteRepository(storage0.getId(), r0_2.getId());
     }
+    
+    @Test
+    public void testUpdatingRepositoryWithNonExistingStorage()
+    {
+        String url = getContextBaseUrl() + "/api/configuration/strongbox/storages/non-existing-storage/fake-repository";
+        RepositoryForm form = new RepositoryForm();
+        
+        givenCustom().contentType(MediaType.APPLICATION_JSON_VALUE)
+        			 .accept(MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Cosmetic: Could you please align the chained method calls like this:
```
        givenCustom().contentType(MediaType.APPLICATION_JSON_VALUE)
                     .accept(MediaType.APPLICATION_JSON_VALUE)
                     .body(form)
                     .when()
                     .put(url)
                     .peek()
                     .then()
                     .statusCode(404);
```

Also, you seem to be using tabs instead of four spaces. Could you please change your code style settings to conform with those of our project? Thanks! :)

For more details, please check out [Coding Convention](https://github.com/strongbox/strongbox/wiki/Coding-Convention).

Reply:
![code-style](https://user-images.githubusercontent.com/14250174/51719488-a79e0b00-2017-11e9-8154-d6a1f8a6968d.PNG)

 I have this coding style applied